OPENEUROPA-2780: Filter out video shot from AV Portal resources. (#44)
openeuropa/media_avportal: README.md

Media AV Portal adds the [European Audiovisual Services](http://ec.europa.eu/avservices/) as a supported media provider.

# Supported media types

> For now we support only 3 media assets types from AV Portal.

Reviewer: ```suggestion
Only 3 media assets types from AV Portal are currently supported:
```
